packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/Chooser.spec.js

```diff
@@ -1,37 +1,75 @@
-import { shallow } from 'enzyme'
-import { types } from 'mobx-state-tree'
 import React from 'react'
+import { render, screen } from '@testing-library/react'
 
-import { default as Task } from '@plugins/tasks/survey'
 import Chooser from './Chooser'
-import FilterStatus from './components/CharacteristicsFilter/FilterStatus'
-import Choices from './components/Choices'
+
+import SurveyTask from '@plugins/tasks/survey'
+import { task } from '@plugins/tasks/survey/mock-data'
+
+const taskWithoutCharacteristics = Object.assign({}, task, { characteristics: {} })
 
 describe('Component > Chooser', function () {
-  let wrapper
-  const task = Task.TaskModel.create({
-    taskKey: 'T0',
-    type: 'survey'
-  })
-  const annotation = task.defaultAnnotation()
-
-  before(function () {
-    wrapper = shallow(
-      <Chooser
-        task={task}
-      />
-    )
-  })
-  
-  it('should render without crashing', function () {
-    expect(wrapper).to.be.ok()
-  })
+  describe('with a survey task with no characteristics', function () {
+    const mockTaskWithoutCharacteristics = SurveyTask.TaskModel.create(taskWithoutCharacteristics)
 
-  it('should render a FilterStatus component', function () {
-    expect(wrapper.find(FilterStatus)).to.have.lengthOf(1)
+    it('should allow you to choose from a list of choices', function () {
+      render(
+        <Chooser
+          task={mockTaskWithoutCharacteristics}
+        />
+      )
+      expect(screen.getByText('Aardvark')).to.be.ok() // first choice per mock-data survey task
+      expect(screen.getByText('Nothing here')).to.be.ok() // last choice per mock-data survey task
+    })
+
+    it('should not allow you to filter the list of choices', function () {
+      render(
+        <Chooser
+          task={mockTaskWithoutCharacteristics}
+        />
+      )
+      expect(screen.queryByText('SurveyTask.CharacteristicsFilter.filter')).to.be.null()
+    })
+
+    it('should not allow you to clear filters on the list of choices', function () {
+      render(
+        <Chooser
+          task={mockTaskWithoutCharacteristics}
+        />
+      )
+      expect(screen.queryByText('SurveyTask.CharacteristicsFilter.clearFilters')).to.be.null()
+    })
   })
 
-  it('should render a Choices component', function () {
-    expect(wrapper.find(Choices)).to.have.lengthOf(1)
+  describe('with a survey task with characteristics', function () {
+    const mockTask = SurveyTask.TaskModel.create(task)
+
+    it('should allow you to choose from a list of choices', function () {
+      render(
+        <Chooser
+          task={mockTask}
+        />
+      )
+      expect(screen.getByText('Aardvark')).to.be.ok() // first choice per mock-data survey task
+      expect(screen.getByText('Nothing here')).to.be.ok() // last choice per mock-data survey task
+    })
+
+    it('should allow you to filter the list of choices', function () {
```

Comment:
Should there also be a user event test that selects a filter and filters the list of choices?

Reply:
I can’t see any tests that check the output of `<Choices />` after `FilterStatus` changes.

Reply:
Great suggestion and I agree that Chooser should have a user event test related to choices after a filter is selected, but I'm running into issues with the filter buttons that I'm working on fixing in #3735 . 

For now I added a test when a filter is provided as a prop to the Chooser component, that confirms the choices are as expected, and I'll add a comment to #3735 to further refactor the Chooser tests with the refactored filtered buttons.